src/utils/helpers/browser/browser.js

```diff
@@ -89,6 +98,16 @@ const Browser = {
   },
 
   /**
+   * Sets the body scroll
+   *
+   * @param {Boolean} enabled
+   * @method setBodyScroll
+   */
+  setBodyScroll: (enabled) => {
+    Browser.getBody().style.overflow = enabled ? '' : 'hidden';
```

Comment:
Yes, adding / removing a class would be better. Which means you could get rid of this function altogether and use something like `Browser.getBody().classList.toggle('body--full-screen-dialog--open')` in the component itself.